src/pypit.py

```diff
@@ -196,7 +195,7 @@ def PYPIT(redname, progname=__file__, quick=False, ncpus=1, verbose=1, logname=N
         initmsgs.error(err.msg, usage=True)
 
     # Execute the reduction, and catch any bugs for printout
-    if debug:
+    if len(debug) > 0:
```

Comment:
If you use a dictionary, this should be:
if debug['develop']:
The purpose of this line is to print out the automatic traceback if someone is developing the code (to see where the bug is). On the other hand, a user that doesn't want to develop the code, just sees a visually friendly "Oops, you found a bug" message!